db/schema.rb

```diff
@@ -32,7 +32,10 @@
   create_table "orders", force: :cascade do |t|
     t.bigint "user_id"
     t.bigint "project_id"
-    t.string "state", default: "pending"
+    t.string "dev_state", default: 0
```

Comment:
Hola Mati! Do you want "dev_state" to be a string or an integer? I'm guessing you want an integer, like this?

```suggestion
    t.integer "dev_state", default: 0
```